### PR TITLE
OCPBUGS-21363: [release-4.13] Bump golang.org/x/net to v0.17.0

### PR DESCRIPTION
**Pull request by awgreene**

### CODE REVIEW

**Review by grokspawn — APPROVED** [review]

**Comment by ncdc** [comment]
/hold

would like to see if we can remove the unrelated dep bumps and have this only be golang/org/x/*

**Comment by awgreene** [comment]
@ncdc fixed this @ncdc, thanks for catching that.

File: vendor/github.com/go-errors/errors/cover.out
Comment (ncdc) [inline vendor/github.com/go-errors/errors/cover.out:1]: I think this needs to be removed?

Comment (awgreene) [reply vendor/github.com/go-errors/errors/cover.out:1]: I think the [verify test](https://github.com/openshift/operator-framework-rukpak/blob/main/Makefile#L93) will fail if I remove this.

Comment (awgreene) [reply vendor/github.com/go-errors/errors/cover.out:1]: Let me check.

Comment (ncdc) [reply vendor/github.com/go-errors/errors/cover.out:1]: It won't. Let's add `cover.out` to `.gitignore`

Comment (awgreene) [reply vendor/github.com/go-errors/errors/cover.out:1]: Tested locally, it worked as it's typically excluded via our `.gitignore` file.

**Review by ncdc — APPROVED** [review]

**Pull request merged**